src/framework/route/helpers.clj

```diff
@@ -0,0 +1,23 @@
+(ns framework.route.helpers
+  (:require
+   [xiana.core :as xiana]))
+
+(defn not-found
+  "Default not-found response handler helper."
+  [state]
+  (xiana/error
+   (-> state
+       (assoc :response {:status 404 :body "Not Found"}))))
+
+(defn action
+  "Default action response handler helper."
+  [{request :request {handler :handler} :request-data :as state}]
+  (try
+    (xiana/ok
+     (assoc state :response (handler request)))
+    (catch Exception e
+      (xiana/error
+       (-> state
+           (assoc :erorr e)
```

Comment:
Typo (:erorr instead of :error)

Reply:
See: https://github.com/Flexiana/framework/pull/88/commits/8115c07cfcafe92b2681baf2840af07a092f7428